deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,223 @@
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: cincinnati-operator
+            spec:
+              containers:
+              - command:
+                - cincinnati-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cincinnati-operator
+                - name: OPERAND_IMAGE
+                  value: quay.io/cincinnati/cincinnati:latest
+                image: quay.io/cincinnati/cincinnati-operator:latest
+                imagePullPolicy: Always
+                name: cincinnati-operator
+                resources: {}
+              serviceAccountName: cincinnati-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - cincinnati-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - cincinnati.openshift.io
+          resources:
+          - '*'
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        serviceAccountName: cincinnati-operator
+    strategy: deployment
+  installModes:
```

Comment:
Not sure what combination we want to go with here.  The OperatorGroup stuff always confuses me.

Reply:
I think the primary advantage of setting `AllNamespaces: true` is that it prevents the operator from being installed more than 1x (assuming it was first subscribed to watching all namespaces). Don't see a specific reason to mark it false. Everything else about these installModes looks legit.